doc/sphinx/addendum/micromega.rst

```diff
@@ -277,7 +277,12 @@ obtain :math:`-1`. By Theorem :ref:`Psatz <psatz_thm>`, the goal is valid.
    :name: zify
 
    This tactic is internally called by :tacn:`lia` to support additional types e.g., :g:`nat`, :g:`positive` and :g:`N`.
-   By requiring the module ``ZifyBool``, the boolean type :g:`bool` and some comparison operators are also supported.
+   Some additional support is provided by the following modules
+
+   + For boolean operators (e.g., `Nat.leb`), require  the module :g:`ZifyBool`.
+   + For comparison operators (e.g., `Z.compare`), require the module :g:`ZifyComparison`.
+   + For native 63 bit integers, require the module :g:`ZifyInt63`.
```

Comment:
Is it sufficient to `Require` the module, or do we need to `Import` it as well?

Reply:
Require is enough.